src/components/PlayerList.jsx

```diff
@@ -0,0 +1,37 @@
+import { Link } from 'react-router-dom';
+import React, { useState, useEffect } from 'react';
+import { getPlayers } from '../services/players.js';
+
+function PlayerList() {
+	const [players, setPlayers] = useState([]);
```

Comment:
all the business logic from line 6 to line 17 needs moved to Players.jsx so it is not living in a component, and the players array should become a prop.